https://github.com/ChilliCream/graphql-platform/pull/7710

diff --git a/Directory.Packages.props b/Directory.Packages.props
index f40edf3d..80c63238 100644
--- a/Directory.Packages.props
+++ b/Directory.Packages.props
@@ -34,7 +34,7 @@
     <PackageVersion Include="R3Extensions.Avalonia" Version="1.2.9" />
     <PackageVersion Include="ReactiveUI" Version="20.1.63" />
     <PackageVersion Include="Spectre.Console.Testing" Version="0.49.1" />
-    <PackageVersion Include="StrawberryShake.Server" Version="14.1.0" />
+    <PackageVersion Include="StrawberryShake.Server" Version="14.2.0-p.7" />
     <PackageVersion Include="System.Linq" Version="4.3.0" />
     <PackageVersion Include="System.Text.Json" Version="9.0.0" VersionOverride="9.0.0" />
     <PackageVersion Include="TextMateSharp.Grammars" Version="1.0.64" />

