--- a/tests/test_run_examples.py
+++ b/tests/test_run_examples.py
@@ -52,12 +52,13 @@ def test_find_run_examples(example: CodeExample, eval_example: EvalExample):
     result = pytester.runpytest('-p', 'no:pretty', '-v')
     result.assert_outcomes(passed=1, failed=1)
 
-    assert result.outlines[-11].startswith('_ _ _ _ ')
-    assert result.outlines[-10:-3] == [
+    assert result.outlines[-12].startswith('_ _ _ _ ')
+    assert result.outlines[-11:-3] == [
         '',
         '    a = 1',
         '    b = 2',
         '>   assert a + b == 4',
+        '           ^^^^^^^^^^',
         'E   AssertionError',
         '',
         'my_file.md:12: AssertionError',

