From cef9276fd1710cc2dad93445683b8155079085bf Mon Sep 17 00:00:00 2001
From: Alfred Wingate <parona@protonmail.com>
Date: Sat, 9 Jul 2022 13:08:07 +0300
Subject: [PATCH] Use Config module for LLD

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 638f39f7e..7aa92be2c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -106,7 +106,7 @@ endif()
 
 find_package(llvm 13)
 find_package(clang 13)
-find_package(lld 13)
+find_package(LLD 13 REQUIRED)
 
 if(ZIG_STATIC_ZLIB)
     list(REMOVE_ITEM LLVM_LIBRARIES "-lz")
@@ -735,7 +735,7 @@ set_target_properties(zigcpp PROPERTIES
 
 target_link_libraries(zigcpp LINK_PUBLIC
     ${CLANG_LIBRARIES}
-    ${LLD_LIBRARIES}
+    ${LLD_EXPORTED_TARGETS}
     ${LLVM_LIBRARIES}
     ${CMAKE_THREAD_LIBS_INIT}
 )
-- 
2.35.1

