https://docs.python.org/3.13/whatsnew/3.13.html#ssl

diff --git a/tests/test_connect.py b/tests/test_connect.py
index 5333e2c..dcb0d09 100644
--- a/tests/test_connect.py
+++ b/tests/test_connect.py
@@ -1689,6 +1689,8 @@ class TestClientSSLConnection(BaseTestSSLConnection):
             ssl.Purpose.SERVER_AUTH,
             cafile=SSL_CA_CERT_FILE,
         )
+        # https://docs.python.org/3.13/whatsnew/3.13.html#ssl
+        ssl_context.verify_flags &= ~ssl.VERIFY_X509_STRICT
 
         with self.assertRaisesRegex(
             exceptions.InvalidAuthorizationSpecificationError,
@@ -1714,6 +1716,8 @@ class TestClientSSLConnection(BaseTestSSLConnection):
                 ssl.Purpose.SERVER_AUTH,
                 cafile=SSL_CA_CERT_FILE,
             )
+            # https://docs.python.org/3.13/whatsnew/3.13.html#ssl
+            ssl_context.verify_flags &= ~ssl.VERIFY_X509_STRICT
             ssl_context.load_cert_chain(
                 CLIENT_SSL_CERT_FILE,
                 keyfile=key_file,
