From 12399aa9280dc7a81abbc7e7fb07a5ecff6fff33 Mon Sep 17 00:00:00 2001
From: Alfred Wingate <parona@protonmail.com>
Date: Fri, 1 Jul 2022 09:09:53 +0300
Subject: [PATCH] CMakeLists.txt: Turn SKIP_TESTS into a proper option

---
 CMakeLists.txt | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ce3f1b5f..28b5030a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -116,6 +116,7 @@ option(SKIP_DESKTOP "Skip building of desktop application" OFF)
 option(BUILD_MAPSHOT "Build mapshot tool" OFF)
 option(USE_PCH "Use precompiled headers" OFF)
 option(NJOBS "Number of parallel processes" OFF)
+option(SKIP_TESTS "Skip tests" OFF)
 
 if (NJOBS)
   message("Number of parallel processes: ${NJOBS}")
@@ -198,10 +199,6 @@ if (PLATFORM_LINUX OR PLATFORM_ANDROID)
   endif()
 endif()
 
-if (NOT SKIP_TESTS)
-  set(SKIP_TESTS FALSE)
-endif()
-
 if (NOT PYTHON_VERSION)
   # FDroid build server has 3.5.3 Python installed.
   set(PYTHON_VERSION 3.5)
-- 
2.35.1

